saleor/graphql/metrics.py

```diff
@@ -0,0 +1,80 @@
+from contextlib import AbstractContextManager
+
+from opentelemetry.semconv._incubating.attributes import graphql_attributes
+from opentelemetry.semconv.attributes import error_attributes
+from opentelemetry.util.types import AttributeValue
+
+from ..core.telemetry import MetricType, Scope, Unit, meter, saleor_attributes
+
+# Initialize metrics
+METRIC_GRAPHQL_QUERY_COUNT = meter.create_metric(
+    "saleor.graphql.query_count",
+    scope=Scope.SERVICE,
+    type=MetricType.COUNTER,
+    unit=Unit.REQUEST,
+    description="Number of GraphQL queries.",
+)
+
+METRIC_GRAPHQL_QUERY_DURATION = meter.create_metric(
+    "saleor.graphql.query_duration",
+    scope=Scope.SERVICE,
+    type=MetricType.HISTOGRAM,
+    unit=Unit.MILLISECOND,
```

Comment:
```suggestion
    unit=Unit.SECOND,
```
https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units
> When instruments are measuring durations, seconds (i.e. s) SHOULD be used.